<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuccessWindow</class>
 <widget class="QWidget" name="SuccessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(82, 82, 82);</string>
  </property>
  <widget class="QLabel" name="label_token">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>831</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_success">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Congratulations, you successfully got a token.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zk">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Zero knowledge proves success.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rand2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(42, 42, 42);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>Waiting to get...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(42, 42, 42);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>Waiting to get...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rand">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
